src/java.base/share/classes/java/lang/reflect/AccessFlag.java

```diff
@@ -457,26 +338,28 @@ public boolean sourceModifier() {
     }
 
     /**
-     * {@return kinds of constructs the flag can be applied to in the
-     * latest class file format version}
+     * {@return kinds of constructs this flag can be applied to in the
+     * current class file format version}
```

Comment:
```suggestion
     * {@return Locations this flag can be applied to in the
     * current class file format version}
```

Reply:
FYI we need to use lowercase in this tag, otherwise rendered output would be 
```
Returns Locations this flag can be applied to in the current class file format version.
```